spark-cassandra-connector/src/main/scala/org/apache/spark/sql/cassandra/TableIdent.scala

```diff
@@ -0,0 +1,6 @@
+package org.apache.spark.sql.cassandra
+
+
+/** Store table identity including table name, keyspace name and option cluster name */
+case class TableIdent(table: String, keyspace: String, cluster: Option[String] = None)
```

Comment:
Why cluster is an Option[String] and keyspace is only a String? Can't we have a default keyspace?